src/kvstore/PartManager.cpp

```diff
@@ -96,25 +96,19 @@ void MetaServerBasedPartManager::onSpaceOptionUpdated(
         "disable_auto_compactions",
         "max_write_buffer_number",
         // TODO: write_buffer_size will cause rocksdb crash
-        // "write_buffer_size",
-        "compression",
+        "write_buffer_size",
         "level0_file_num_compaction_trigger",
         "level0_slowdown_writes_trigger",
         "level0_stop_writes_trigger",
         "target_file_size_base",
         "target_file_size_multiplier",
         "max_bytes_for_level_base",
         "max_bytes_for_level_multiplier",
-        "ttl",
-        "block_size",
-        "block_restart_interval"
     };
     static std::unordered_set<std::string> supportedDbOpt = {
         "max_total_wal_size",
         "delete_obsolete_files_period_micros",
         "max_background_jobs",
```

Comment:
support sub_maxcompactions

Reply:
it can't be changed dynamically, for those options, it would take effect when reboot.